editoast/src/views/search.rs

```diff
@@ -683,6 +691,48 @@ pub(super) struct SearchResultItemScenario {
     tags: Vec<String>,
 }
 
+#[derive(Search, Serialize, ToSchema)]
+#[search(
+    table = "train_schedule",
+    column(name = "timetable_id", data_type = "integer"),
+    column(name = "train_name", data_type = "string")
+)]
+#[allow(unused)]
+#[derive(serde::Deserialize)]
```

Comment:
style nitpick:

```suggestion
#[derive(Search, Serialize, Deserialize, ToSchema)]
#[search(
    table = "train_schedule",
    column(name = "timetable_id", data_type = "integer"),
    column(name = "train_name", data_type = "string")
)]
#[allow(unused)]
```

or:

```suggestion
#[derive(Search, Serialize, ToSchema)]
#[cfg_attr(test, derive(serde::Deserialize))]
#[search(
    table = "train_schedule",
    column(name = "timetable_id", data_type = "integer"),
    column(name = "train_name", data_type = "string")
)]
#[allow(unused)]
```

(only derives the implementation when `test` is enabled since we don't need it otherwise)